### PR TITLE
DOC-2177: document `picker_text` property for `urlinput` dialog component

### PR DESCRIPTION
**Pull request by brian-forte**
DOC-2177: document `picker_text` property for `urlinput` dialog component

Changes:
* Added documentation of the `picker_text` property to the `urlinput` dialog section of `dialog-components.adoc`.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [ ] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/dialog-components.adoc [hunk 1]
```diff
@@ -446,6 +446,10 @@ A *listbox* is a composite component with a label and a dropdown list of options
 }
 ----
 
+[source,js]
```
Comment (mkzhx) [inline modules/ROOT/pages/dialog-components.adoc:449]: Is this addition intentional?

File: modules/ROOT/pages/dialog-components.adoc [hunk 2]
```diff
@@ -669,23 +673,67 @@ A *urlinput* is a specialized composite component for URL input or file upload.
...
+Whether the component should initially be disabled.
+
+|picker_text
+|string
+|optional
+|Sets an alternative tooltip for the file picker button.
+
+If `picker_text` is not specified, the tooltip defaults to the value provided by the `label` property.
+
+The `label` property supplies one of two default values for the tooltip: *Browse files* or *Browse links*.
```
Comment (mkzhx) [inline modules/ROOT/pages/dialog-components.adoc:727]: We don't have default values so let's remove this line.

File: modules/ROOT/pages/dialog-components.adoc [hunk 3]
```diff
@@ -695,6 +743,7 @@ The filepicker will accept any file type and the typeahead will include 5 previo
   filetype: 'file', // allow any file types
   label: 'Url', // text for component label
   enabled: false // enabled state
```
Comment (mkzhx) [inline modules/ROOT/pages/dialog-components.adoc:745]: Comma to separate object properties
```suggestion
  enabled: false, // enabled state
```

**Review by mkzhx — APPROVED** [review]

File: modules/ROOT/pages/dialog-components.adoc (hunk 2 above)
Comment (brian-forte) [inline modules/ROOT/pages/dialog-components.adoc:727]:
```suggestion
```

**Review by kemister85 — APPROVED** [review]

File: modules/ROOT/pages/dialog-components.adoc (hunk 1 above)
Comment (brian-forte) [reply modules/ROOT/pages/dialog-components.adoc:449]: Good catch. Looks like a code sample boilerplate inadvertently left in place. I’ll remove it.

File: modules/ROOT/pages/dialog-components.adoc [hunk 4]
```diff
@@ -446,6 +446,10 @@ A *listbox* is a composite component with a label and a dropdown list of options
 }
 ----
 
+[source,js]
+----
+----
```
Comment (brian-forte) [inline modules/ROOT/pages/dialog-components.adoc:451]:
```suggestion
```
